src/status_im/test_runner.cljs

```diff
@@ -0,0 +1,104 @@
+(ns status-im.test-runner
+  {:dev/always true}
+  (:require
+   [cljs.test :as ct]
+   [clojure.string :as str]
+   [shadow.test :as st]
+   [shadow.test.env :as env]))
+
+(defonce repl? (atom false))
```

Comment:
`@repl?` is `true` when running with `--repl` flag, `node --require ./test-resources/override.js target/test/test.js --repl`